### PR TITLE
tools: Add Akash deployment configuration

### LINKED ISSUE #1378: Akash deployment configuration template
- [ ] blocked by https://github.com/sourcenetwork/defradb/issues/1355
- [ ] blocked by #1266 

related documentation issue: https://github.com/sourcenetwork/docs.source.network/issues/100

because there is no ACL currently, explain how to use specifically as part of a stack with multiple containers and expose whitelist

TBD SSL certificates

- [ ] make PR on https://github.com/akash-network/deploy-templates to publish defradb's workflow

### LINKED ISSUE #1659: Akash deployment

### PR DESCRIPTION
**Pull request by nasdf**
## Relevant issue(s)

Resolves #1659
Resolves #1378 

## Description

This PR adds an Akash deployment template to DefraDB `tools/cloud/akash`.

Akash deployment URI: http://0i8se95qgletbeq16atqulhhc8.ingress.midwesternwireless.com/api/v0

## Tasks

- [x] I made sure the code is well commented, particularly hard-to-understand areas.
- [x] I made sure the repository-held documentation is changed accordingly.
- [x] I made sure the pull request title adheres to the conventional commit style (the subset used in the project can be found in [tools/configs/chglog/config.yml](tools/configs/chglog/config.yml)).
- [x] I made sure to discuss its limitations such as threats to validity, vulnerability to mistake and misuse, robustness to invalidation of assumptions, resource requirements, ...

## How has this been tested?

Manual deployment via Cloudmos.

Specify the platform(s) on which this was tested:
N/A

### CODE REVIEW

File: tools/cloud/akash/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        storage:
+          size: 1Gi
+  placement:
+    akash:
+      attributes:
+        host: akash
+      signedBy:
+        anyOf:
+          - "akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63"
+          - "akash18qa2a2ltfyvkyj0ggj3hkvuj6twzyumuaru9s4"
```
Comment (shahzadlone) [inline tools/cloud/akash/deploy.yaml:41]: question: Where are these signatures from?

File: tools/cloud/akash/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# DefraDB
+
+DefraDB is a user-centric database that prioritizes data ownership, personal privacy, and information security. Its data model, powered by the convergence of [MerkleCRDTs](https://arxiv.org/pdf/2004.00107.pdf) and the content-addressability of [IPLD](https://docs.ipld.io/), enables a multi-write-master architecture. It features [DQL](https://docs.source.network/references/query-specification/query-language-overview), a query language compatible with GraphQL but providing extra convenience. By leveraging peer-to-peer networking it can be deployed nimbly in novel topologies. Access control is determined by a relationship-based DSL, supporting document or field-level policies, secured by the SourceHub network. DefraDB is a core part of the [Source technologies](https://source.network/) that enable new paradigms of decentralized data and access-control management, user-centric apps, data trustworthiness, and much more.
+
+Read the documentation on [docs.source.network](https://docs.source.network/).
```
Comment (AndrewSisley) [inline tools/cloud/akash/README.md:5]: suggestion: A link to `https://github.com/sourcenetwork/defradb` might be handy too.

**Review by AndrewSisley — APPROVED** [review]
LGTM, added a quick/optional suggestion to the readme

File: tools/cloud/akash/deploy.yaml (hunk 1 above)
Comment (nasdf) [reply tools/cloud/akash/deploy.yaml:41]: I asked for clarification on these in the Akash discord.

- `akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63` Akash community audits
- `akash18qa2a2ltfyvkyj0ggj3hkvuj6twzyumuaru9s4` moultrie-audits

There is no official registry for Akash audit addresses and these two seem to appear in most Akash configs.

**Comment by nasdf** [comment]
> Looks alright, however I don't know much about Akash configs haha, is there a test run for this? Or list of steps I can take to test it myself?

The process requires creating an Akash account, setting up a certificate, and having at a minimum 5 AKT.  I believe we are going to add a guide to our documentation. I'm happy to do a walkthrough for anyone interested.

**Pull request merged**